Basics/Git.md

```diff
@@ -4,50 +4,130 @@
+* **fix**
+  * used for fixing non-critical production bugs or bugs occurred in development phase
+  * has prefix ***fix/***, e.g. fix/123-wrong-cards-sorting
+* **refactor**
+  * used for codebase/project refactoring
+  * Has prefix ***refactor/***, e.g. refactor/123-alert-and-action-sheet
+* **hotfix**
+  * used for fixing critical bugs in production
+  * may be branched off from the corresponding tag and create PR to the ***master*** or to the latest ***release*** branch if one exists
+  * has prefix ***hotfix/***, e.g. hotfix/1337-recipe-details-crash
+  * Example usage: there is a bug on app version 1.2.0. In that case you checkout to a commit tagged with v1.2.0. You create release/1.2.1 branch from that tag, and you create hotfix/whatever from that branch where you implement the fix and create PR back to the release/1.2.1 branch
+ 
+#### Pull requests
+Each pull request must have an automatic check - running tests or just building the app if your project doesn’t have tests. Setup on GitHub is described in general rules of this handbook, while on how to set up Bitrise Checks on projects you have [Android][7] and [iOS][8] handbook chapters.
+
+#### Deployment
+Deployment is done via tags and Bitrise CI/CD. Here are the chapters on setting up Bitrise on your project: [Android][7] and [iOS][9].
+
+Sync with another platform as much as possible. Few stuff that should be defined and synced:
+* Environment names - staging, production, uat, beta… whatever is convention on project, but should be the same. Exception probably will be iOS with its production and app store environments due to technical limitations. If using TryOutApps, please sync environment names.
+* Build number/code version - Must be numbers only value. For build number you can use commit count (could be fetched with `git rev-list HEAD --count` from deploy script and bitrise has global variable `$GIT_CLONE_COMMIT_COUNT` that could be used) or manually auto increment number value (i.e. before creating a new build, last build number should be fetched and increment by 1).
+* Tag names - tags (used for tagging the TryOutApps version and/or triggering Bitrise CI) should be comprised of three parts: environment, app version with ***v*** prefix and build number/code version, e.g. ***internal-staging/v1.2.3-1234***
+  * Each project will have different environments so this will be heavily project dependent, but some suggestions: tags should have prefix like ***internal-staging/***, ***internal-production/***, ***appstore/***... Due to limitation on tag triggers on Bitrise, tags with multiple / are not permitted, so you should use something like ***internal-production/***, ***internal-staging/***.
+  * Use ***internal-*** prefix for all tags which will trigger internal builds (Labs, TryOutApps). Reason behind it: internal tags are not so relevant and one should clear internal tags from time to time to keep the project clean (LEs will prepare the script for it).
```

Comment:
```suggestion
  * Use ***internal-*** prefix for all tags which will trigger internal builds on TryoutApps. The reason behind it: internal tags are not so relevant and one should clear internal tags from time to time to keep the project clean.
```

Officially, Labs name is deprecated and we should only use "TryoutApps" naming 🤷🏽‍♂️